### PR TITLE
Cleanup code

### PR DESCRIPTION
**Pull request by pubudu91**
## Purpose
Cleans up the unused/unnecessary code and modules.

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -34,7 +34,6 @@ include(':stdlib')
 //include(':ballerina-crypto')
```
Comment (MaryamZi) [inline settings.gradle:34]: We should probably remove what's commented out too, right? Maybe we can do that in a separate PR.

**Review by hasithaa — DISMISSED** [review]

File: stdlib/http/build.gradle [hunk 2]
```diff
@@ -1,122 +0,0 @@
...
-    interopImports project(':ballerina-runtime-api')
-    interopImports project(':ballerina-mime')
-
-    // transitive
-    balaImplementation project(path: ':ballerina-task', configuration: 'balaImplementation')
-
-    interopImports project(':ballerina-io')
-
-    implementation 'io.netty:netty-codec-http'
-    implementation ('org.wso2.transport.http:org.wso2.transport.http.netty') {
```
Comment (warunalakshitha) [inline stdlib/http/build.gradle:91]: There are some netty dependencies in io.netty:netty-codec-http javaProject.gradle file. Shall we remove them too?

Comment (pubudu91) [reply stdlib/http/build.gradle:91]: Ack. Will remove and see.

Comment (warunalakshitha) [reply stdlib/http/build.gradle:91]: We can fix with a separate PR as well.

File: compiler/ballerina-lang/src/test/java/org/wso2/ballerinalang/compiler/BLangNodeTransformerTest.java [hunk 3]
```diff
@@ -280,16 +246,12 @@ private static JsonObject fileContentAsObject(String filePath) {
...
-                }
-            }, true, "UTF-8");
-        }
+    private static Project loadProject(String sourceFilePath) {
+        Path sourcePath = Paths.get(sourceFilePath);
+        String sourceFileName = sourcePath.getFileName().toString();
+        Path sourceRoot = RES_DIR.resolve(sourcePath.getParent());
+        Path projectPath = Paths.get(sourceRoot.toString(), sourceFileName);
+        BuildOptionsBuilder buildOptionsBuilder = new BuildOptionsBuilder();
+        return ProjectLoader.loadProject(projectPath, buildOptionsBuilder.build());
```
Comment (azinneera) [inline compiler/ballerina-lang/src/test/java/org/wso2/ballerinalang/compiler/BLangNodeTransformerTest.java:255]: We can use `ProjectLoader.loadProject(projectPath)`. It will set the default build options.

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/util/SQLDBUtils.java [hunk 4]
```diff
@@ -18,6 +18,7 @@
 package org.ballerinalang.test.util;
```
Comment (MaryamZi) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/util/SQLDBUtils.java:18]: IINM, none of the tests that use this class seem to be running. Shall we create an issue to check if they are still valid/belong in the lang repo and enable/remove these.

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/packageimport/PackageImportTest.java [hunk 5]
```diff
@@ -93,7 +93,7 @@ public void testUnusedImports() {
         BAssertUtil.validateError(result, i, "unused import module 'ballerina/jballerina.java as otherJAVA'", 2, 1);
     }
 
-    @Test
+    @Test(enabled = false)
```
Comment (MaryamZi) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/packageimport/PackageImportTest.java:96]: Why are we disabling this?

File: stdlib/http/build.gradle (hunk 2 above)
Comment (pubudu91) [reply stdlib/http/build.gradle:91]: I think we'll need to take a closer look at these. Seems to be in used in tests and test utils.

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/packageimport/PackageImportTest.java (hunk 5 above)
Comment (pubudu91) [reply tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/packageimport/PackageImportTest.java:96]: Need to think of a different way to write this. This uses `ballerina/io` to print a message in module init function and then capture that to a buffer and compare the outputs to verify the behaviour. With the `ballerina/io` module removed, can't do it that way. So disabled it for now, will create an issue for this.

Comment (MaryamZi) [reply tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/packageimport/PackageImportTest.java:96]: We can probably use interop similar to https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-unit-test/src/test/resources/test-src/execution/proj8/main.bal#L18.

Shall we create an issue if we are disabling this?

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/util/SQLDBUtils.java (hunk 4 above)
Comment (pubudu91) [reply tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/util/SQLDBUtils.java:18]: Ack. Will create an issue for this.

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/packageimport/PackageImportTest.java (hunk 5 above)
Comment (pubudu91) [reply tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/packageimport/PackageImportTest.java:96]: Yeah, a util like this should work fine. Will address this separately.

**Review by azinneera — APPROVED** [review]

**Review by MaryamZi — APPROVED** [review]
